experiments/run_benchmarks_experiments.py

```diff
@@ -37,7 +37,7 @@
             "--cdl-root",
             CDL_DATA_ROOT,
             "--num-workers",
-            "6",
+            "8",
```

Comment:
My VM has 12 cores, but 8 workers provides the best performance since our batch sizes are powers of 2.